src/widget/wvumeter.cpp

```diff
@@ -238,3 +248,33 @@ void WVuMeter::paintEvent(QPaintEvent *) {
     m_dLastParameter = m_dParameter;
     m_dLastPeakParameter = m_dPeakParameter;
 }
+
+QSize WVuMeter::sizeHint() const {
+    QStyleOption option;
+    option.initFrom(this);
+    
+    QSize widgetSize = WWidget::sizeHint();
```

Comment:
What is the default WWidget::sizeHint()? 
What happens if the background is smaller than the VU Pixamp?